src/ts/state/reducer.ts

```diff
@@ -92,6 +93,14 @@ export interface IBlockInFlight {
   block: IBlock;
 }
 
+/**
+ * A consumer-supplied predicate that allows consumers to ignore matches.
+ * Handy when, for example, consumers know that parts of the document are
+ * exempt from checks.
+ */
+export type IIgnoreMatch = (match: IMatch) => boolean;
+export const defaultIgnoreMatch: IIgnoreMatch = () => false;
```

Comment:
You're going to hate me, but I don't like the name.  Why not 'includeAll' or something with business meaning?

Reply:
> Did you?
I'm still blaming the heat 😅 

> You're going to hate me, but I don't like the name. Why not 'includeAll' or something with business meaning?
Naw, I completely agree. Like you say, business meaning > what the thing _is_. Just changed 👍